frontend/portals/MaintenanceMode/index.jsx

```diff
@@ -1,6 +1,10 @@
 import appConfig from '@shopgate/pwa-common/helpers/config';
 import I18n from '@shopgate/pwa-common/components/I18n';
+import { Link } from '@shopgate/engage/components';
+import Button from '@shopgate/pwa-ui-shared/Button';
```

Comment:
the Button import can also come from the engage library so you can add it to the Link import
`import { Link, I18n } from '@shopgate/engage/components';`